visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.html

```diff
@@ -21,12 +21,15 @@
 		>
 
 		<md-button
-			ng-if="$ctrl._viewModel.codeMapNode.visible"
+			ng-if="!$ctrl._viewModel.codeMapNode.isBlacklisted"
 			ng-click="$ctrl.flattenNode()"
 			title="Flatten node and children-nodes, but keep an empty space"
 			><i class="fa fa-eye-slash"></i> Flatten</md-button
 		>
-		<md-button ng-if="!$ctrl._viewModel.codeMapNode.visible" ng-click="$ctrl.showNode()" title="Show node and children-nodes"
+		<md-button
+			ng-if="$ctrl._viewModel.codeMapNode.isBlacklisted === 'exclude'"
+			ng-click="$ctrl.showNode()"
+			title="Show node and children-nodes"
 			><i class="fa fa-eye"></i> Show</md-button
 		>
```

Comment:
I think this is a dead button, which is never displayed

Reply:
It's not. It was just not working before. Now it is again. :)

Reply:
flatten a node, go to treeview and rightclick it again to "show" it